test/passing/tests/break_infix.ml.ref

```diff
@@ -0,0 +1,87 @@
+let _ =
+  get_succs parent
+  |> Sequence.of_list
+  |> Sequence.filter ~f:(fun n -> not (equal node n))
+  |> Sequence.Generator.of_sequence
+
+let git_clone ~branch ~remote ~output_dir =
+  run_and_log
+    Cmd.(
+      v "git"
+      % "clone"
+      % "--depth"
+      % "1"
+      % "--branch"
+      % branch
+      % remote
+      % p output_dir )
```

Comment:
This is vertical because of the applications of `v` and `p`, it's correct but annoying. 1 char long function names are intended to be discreet, like an prefix operator.
Would it make sense to add an exception this case ?

Reply:
It makes sense but it should be done separately, considering this kind of function application as simple would be useful in other contextes.

Reply:
I don't think it should be done separately. There's a risk of not implementing it due to timing and finding that generalizing it is too breaking.
This option needs a different "simple" predicate that can be tweaked.

Reply:
This part of the algorithm doesn't rely on whether an expression is simple but on the precedence, we can tweak it to treat this kind of application differently but I'm afraid it would look weird (what about `f (fun x -> complex expr)`). I think it's reasonable to keep the current behavior until users complain too much, it's gated behind a new option anyway.